src/components/forms/DiskSizeSelector.tsx

```diff
@@ -76,12 +76,11 @@ const DiskSizeSelector: FC<Props> = ({
           title={disabledReason}
         />
       </div>
-      {(help || helpTotal) && (
-        <p className="p-form-help-text">
-          {help}
-          {helpTotal}
-        </p>
-      )}
+      <p className="p-form-help-text">
+        {help}
+        {help && helpTotal && " "}
+        {helpTotal}
+      </p>
```

Comment:
```suggestion
      <p className="p-form-help-text">
        {[help, helpTotal].filter(Boolean).join(' ')}
      </p>
```

More elegant but harder to read. Wdyt ?

Reply:
@edlerd suggested to use a `<br />` between the two texts.

Reply:
Yeah, I saw afterwards. That breaks my elegant solution ! 🫩